UNRELEASED-V4.md

```diff
@@ -6,25 +6,30 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
-- `ChoiceList` `title` is required for accessibility. It can be hidden with `titleHidden`. ([#1575](https://github.com/Shopify/polaris-react/pull/1575))
-- Remove the WithRef component. This was never documented and was intended for internal use only but was part of our public API ([#1610](https://github.com/Shopify/polaris-react/pull/1610)).
-- Remove the WithContext component. This was never documented and was intended for internal use only but was part of our public API ([#1641](https://github.com/Shopify/polaris-react/pull/1641)).
+- Removed `groups` prop on `Select`. Pass groups to the `options` prop instead. ([#1830](https://github.com/Shopify/polaris-react/pull/1830))
+- Removed `Autocomplete.ComboBox.TextField` and `Autocomplete.ComboBox.OptionList`. You should use the `Autocomplete.TextField` and `OptionList` components instead. ([#1830](https://github.com/Shopify/polaris-react/pull/1830))
+- Removed `secondaryFooterAction` prop on `Card`. Pass an array of secondary actions to the `secondaryFooterActions` prop instead. ([#1830](https://github.com/Shopify/polaris-react/pull/1830))
+- Removed `iconBody` prop on `Navigation`. Pass a string into the `icon` prop instead. ([#1830](https://github.com/Shopify/polaris-react/pull/1830))
+- Removed the WithContext component. This was never documented and was intended for internal use only but was part of our public API ([#1641](https://github.com/Shopify/polaris-react/pull/1641)).
+- Removed the WithRef component. This was never documented and was intended for internal use only but was part of our public API ([#1610](https://github.com/Shopify/polaris-react/pull/1610)).
 - Removed support for passing a string into `<Icon source>` to load a bundled icon. You must load the required icon directly from `@shopify/polaris-icons` instead ([#1604](https://github.com/Shopify/polaris-react/pull/1604)).
 - Removed support for passing a "SvgSource" shaped object into `<Icon source>` to load an icon imported using Shopify's legacy icon loader. You must update sewing-kit to at least v0.82.0 which replaced the legacy loader with using SVGR ([#1604](https://github.com/Shopify/polaris-react/pull/1604)).
 - Removed support for passing a React Element into `<Icon source>`. You must pass in a React Component that returns an SVG element instead. ([#1604](https://github.com/Shopify/polaris-react/pull/1604)).
 - Removed support for `<Icon untrusted>`. Passing a string into `source` will now always load an untrusted icon, you don't need that additional property. ([#1604](https://github.com/Shopify/polaris-react/pull/1604)).
+- Removed `Navigation.UserMenu`. Use `TopBar.UserMenu` instead ([#1599](https://github.com/Shopify/polaris-react/pull/1599))
+- `ChoiceList` `title` is required for accessibility. It can be hidden with `titleHidden`. ([#1575](https://github.com/Shopify/polaris-react/pull/1575))
```

Comment:
```suggestion
- Made `ChoiceList`’s `title` prop required to improve accessibility. It can be hidden with `titleHidden`. ([#1575](https://github.com/Shopify/polaris-react/pull/1575))
```